_maps/shuttles/emergency_rollerdome.dmm

```diff
@@ -284,7 +284,7 @@
 /turf/open/floor/wood,
 /area/shuttle/escape)
 "RN" = (
-/obj/machinery/jukebox/disco/indestructible,
+/obj/machinery/jukebox/disco,
```

Comment:
i do not think this is a good change because it merely obfuscates the fact that disco machines lag out the server. i would much prefer to see someone attempt to correct this lag before this mapping change is made.

i am amenable to everything else that isn't mentioned in this PR.